### ISSUE TITLE
How to install angular-cli inside restricted Network access environment

### ISSUE THREAD

**Issue by Sankey28** [angular/angular-cli#8999]
### Versions
```
<!--
Output from: `ng --version`.
If nothing, output from: `node --6.11.3` and `npm --3.10.10`. Windows 7
-->
```

### Issue
I work in the company where I have restricted environment. If I need to install angular-cli on my machine, I can request network administrator to unblock the URL. But at first place, I don't know which URL to unblock.
* Please provide URL which are needs to unblocked for successful installation of angular-cli.
* If possible provide a GET STARTED guide for the enterprise environment.

I have unblocked the URL https://registry.npmjs.org, but it is not helping.

![error](https://user-images.githubusercontent.com/24712341/34355394-e2f07b96-ea5a-11e7-9ec0-c5cfd53fe5f6.PNG)
[npm-debug.log](https://github.com/angular/angular-cli/files/1587221/npm-debug.log)

**Comment by Sankey28** [comment]
I have configured the Proxy as
• npm set proxy=http://host:port
• npm set https-proxy=http://host:port

After that If you try ng -v it will give error as there is not PATH variable configured.
To counter this, use Node.js command prompt to navigate to your folder and start working with Angular.

**Comment by filipesilva** [comment]
This seems like a problem with your node/npm and not with Angular CLI. 

Please have a look at the [fixing npm permissions page](https://docs.npmjs.com/getting-started/fixing-npm-permissions), [common errors page](https://docs.npmjs.com/troubleshooting/common-errors), [npm issue tracker](https://github.com/npm/npm/issues), or open a new issue in that issue tracker if the problem you are experiencing isn't known.

**Comment by Charan29** [comment]
Hi @Sankey28 , I am also facing proxy config issue. Did you problem solved ?.
